### DISCUSSION TITLE
Birt 4.14 :  run report in browser does not work with birt-report-designer-all-in-one-4.14.0-202312020807-win32.win32.x86_64

### DISCUSSION THREAD

**Discussion by mabu999** [eclipse-birt/birt#1511] [Q&A] [answered]
Hello Community ,

I was just testing the new Version 4.14 (birt-report-designer-all-in-one-4.14.0-202312020807-win32.win32.x86_64).
When I try to run a Report in Browser that does not work, with 4.13 everthing works fine.

Then I was looking for the Problem. 

I figured out, that the port on localhost is not open,

For instance : The url is : 
http://127.0.0.1:60654/viewer/frameset?__report=%5C%5CEFLV0001%5CIPES%5CAIX%5CEclipse_workspace_4.14%5CAIX%5COnline%5C63-AIX_CPU_Physical_d.rptdesign&__format=html&__svg=true&__locale=de_DE&__timezone=ECT&__masterpage=true&__rtl=false&__cubememsize=10&__resourceFolder=%5C%5CEFLV0001%5CIPES%5CAIX%5CEclipse_workspace_4.14%5CAIX&__parameterpage=true&110716252

Then I did a netstat -an | grep 60654 in CMD
but this port was not listening to anything.

Does anybody has the same Problem ?

Any Ideas ?

**Comment by speckyspooky** [comment]
I have tested under Windows 11 the same BIRT 4.14 version like you and my testing reports are working in the viewer.
My port is a completely different port like yours, in my case: "http://127.0.0.1:56260/viewer/frameset?__report=..." 

But all works fine.

**Reply by mabu999** [reply]
I cannot imagine that it's only a question of the Windows Version ?

**Comment by mabu999** [comment]
I use win10, I have no Win 11

with every new request you get a new port in the browser.

Result Birt 4.14 


![birt2](https://github.com/eclipse-birt/birt/assets/122016135/2bc4d2b0-31b1-4dba-93f4-93eccde6cf24)

![birt1](https://github.com/eclipse-birt/birt/assets/122016135/4e62df4f-17ff-4972-8359-512c86e079cc)

Result Birt 4.13

![birt3](https://github.com/eclipse-birt/birt/assets/122016135/0acf5bc4-3078-4bba-bb19-09e0d0b2f35c)

**Reply by mabu999** [reply]
Here my detailed findings from command line :


## birt Version 4.13

# find birt Process ID
C:\WINDOWS\system32>tasklist | findstr birt
birt.exe                     17912 Console                    1       607.156 K

# Netstat for Process ID 17912
C:\WINDOWS\system32>netstat -b -a -n -o | findstr 17912
  TCP    127.0.0.1:64713        0.0.0.0:0              ABHÖREN         17912
# Process ID 17912 opens Port 64713

# Netstat for Port 64713
C:\WINDOWS\system32>netstat -b -a -n -o | findstr 64713
  TCP    127.0.0.1:64363        127.0.0.1:64713        WARTEND         0
  TCP    127.0.0.1:64364        127.0.0.1:64713        WARTEND         0
  TCP    127.0.0.1:64368        127.0.0.1:64713        HERGESTELLT     20364
  TCP    127.0.0.1:64713        0.0.0.0:0              ABHÖREN         17912
  TCP    127.0.0.1:64713        127.0.0.1:64368        HERGESTELLT     17912
# --> ok

## Birt 4.14

# find birt Process ID
C:\WINDOWS\system32>tasklist | findstr birt
birt.exe                     16708 Console                    1       452.884 K

# Netstat for Process ID 16708
C:\WINDOWS\system32>netstat -b -a -n -o  | findstr 16708
# Process ID 16708 has no open Port
# --> not ok

**Reply by mabu999** [reply]
I just tested the same report with birt-rcp-report-designer-4.14.0-202312020807, everything works fine

**Reply by mabu999** [reply]
I just tested with the newest Version (birt-report-designer-all-in-one-**4.15.0-202312050850**) and the same result, connection error in browser and no port openen in CMD

I think it must be a bug

**Reply by speckyspooky** [reply]
Sorry for that, but I cannot confirm currently your issue.
I tested the BIRT 4.14 & BIRT 4.15 with version all-in-one-designer like you and I got the viewer component.
I used the same CDM and all is fine.

Yes, I can see that your "netstat" is negative but I have no hint why your port sould be closed or not opened.

Do you have some hints on eclipse log into the UI one the command line or the log-file on the eclipse folder?

Could you please try also the "eclipse.exe" of the all-in-one-designer because there are 2 exe-files "eclipse.exe" and "birt.exe".
Both versions are running on my side. Therefore it would be good to know if the issue is given on your side on both executables.

**Reply by mabu999** [reply, marked as answer]
It seems that's only my personal problem. 

I installed Birt on another PC (PC2) with Java 10.0.2, everthing works without any trouble.
I used Java 17.0.2 for my first testing on PC1. No I Changed my Path to 10.0.2 on PC1 and Birt crashed with "Could not resolve module: org.eclipse.jetty.ee8.plus" in the log.
Last but not least I changed the Path to 1.8.0_192, Birt started but the Ports for Report viewing are still closed.

I tested both, birt.exe + eclipse.exe with this Java Versions with different results but without open ports

What is the Best way to force Birt in Debugmode to get logfiles for further debuging ?
